### PR TITLE
[Needs Review Before Approval] add tallercentralwiki on wgExtraInterlanguageLinkPrefixes

### PR DESCRIPTION
**Pull request by ucronistaw**
**[Needs Review Before Approval]**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1704,6 +1704,9 @@
 	'wgInterwikiCentralDB' => array(
 		'default' => 'metawiki',
 	),
+	'wgExtraInterlanguageLinkPrefixes' => array(
+		'tallercentralwiki' => voy,
```
Comment (Amanda-Catherine) [inline LocalSettings.php]: The word “voy” needs to be enclosed with an apostrophe on both sides

Comment (ucronistaw) [reply LocalSettings.php]: uh, anything else?

**Comment by JohnFLewis** [comment]
What’s a defaul?

File: LocalSettings.php [hunk 2]
```diff
@@ -1704,6 +1704,12 @@
 	'wgInterwikiCentralDB' => array(
 		'default' => 'metawiki',
 	),
+	'wgExtraInterlanguageLinkPrefixes' => array(
+		'defaul' => array(),
```
Comment (paladox) [inline LocalSettings.php]: misspelt.

**Pull request merged**